lib/active_merchant/billing/check.rb

```diff
@@ -7,8 +7,8 @@ module Billing #:nodoc:
     # You may use Check in place of CreditCard with any gateway that supports it.
     class Check < Model
       attr_accessor :first_name, :last_name,
-        :bank_name, :routing_number, :account_number,
-        :account_holder_type, :account_type, :number
+                    :bank_name, :routing_number, :account_number,
+                    :account_holder_type, :account_type, :number
```

Comment:
To be honest, I'm not really a fan of all these changes (nor Rubocop's pedantic standards around this). It makes this PR pretty huge. Not sure if we ever discussed which rules we want to adopt or not - but the default settings don't seem to apply to the existing code style.